Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -45,15 +51,24 @@ public function configureFormFields(FormMapper $formMapper)
             'required' => false,
         );
 
-        $formMapper->with($this->formGroup, array(
-            'translation_domain' => 'CmfCoreBundle',
-            ))
-            ->add('publish_start_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_start_date',
-            ))
-            ->add('publish_end_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_end_date',
+        $formMapper
+            ->with($this->formTab, array_merge(
+                array('tab' => true), // use option instead of method to be compatible with Sonata <2.3
+                'form.tab_publish' === $this->formTab
+                    ? array('translation_domain' => 'CmfCoreBundle')
+                    : array()
```

Comment:
why? should we not always use our translation domain? otherwise i can not move things to a different tab without also having to copy translations to the default domain.